### PR TITLE
Selected Alarmtone Playing

### PR DESCRIPTION
**Pull request by sammy1997**
Fixes issue 
#249 

### Screenshots or GIF -
N.A.

### CODE REVIEW

**Comment by sammy1997** [comment]
@anubhakushwaha   please review this. Thanks.

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.peacecorps.malaria.services;
+    package com.peacecorps.malaria.services;
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: Please don't insert unnecessary spaces

File: .idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (anubhakushwaha) [inline .idea/gradle.xml]: Don't commit these files please

**Comment by anubhakushwaha** [comment]
@sammy1997 Hey, could you rebase your work to current head in develop and further try to limit the number of commits on basis of overall functionality added, thanks. Let me know incase of anything else.

**Comment by anubhakushwaha** [comment]
@sammy1997 Further could you send a separate PR for just the upgraded gradle file?

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return START_NOT_STICKY;
+    }
+
+    @Override
+    public void onDestroy()
+    {
+        Log.e("RingService OnEnd", "Stopping Song");
+        ringtone.stop();
+    }
+}
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: Further be consistent with the rest of the codebase, in the sense that place the curly braces on the same line.

Comment (sammy1997) [reply malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: @anubhakushwaha ma'am, can you please tell me which line the curly braces are out of line?

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+public class RingService extends Service {
+    private Ringtone ringtone;
+
+    public RingService() {
+    }
+
+    @Override
+    public IBinder onBind(Intent intent)
+    {
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: Here , it should be of form:
func{
.....
}

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        return null;
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId)
+    {
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: Same

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        Log.e("RingService OnStart","Starting Song");
+        this.ringtone = RingtoneManager.getRingtone(this, ringtoneUri);
+        ringtone.play();
+
+        return START_NOT_STICKY;
+    }
+
+    @Override
+    public void onDestroy()
+    {
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: Same

**Comment by sammy1997** [comment]
@anubhakushwaha  I have made the changes that you requested. Please review it again. I have re-based the code as well. Thanks

File: malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java (hunk 5 above)
Comment (sammy1997) [reply malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java]: @anubhakushwaha Please check the commit 19b22b9e14c041060f1f17ad512a209ac324edd6 (Start and stop ring service) as I have squashed all the commits with minor changes into that one( as you wanted commits according to functionality)

**Comment by anubhakushwaha** [comment]
@sammy1997  Hey I think you have pulled the master branch work into your remote develop because of which it has the commits of master branch as well, please take of that. I would suggest either undoing your commits on your remote develop or delete the develop branch, create it again and pull the develop branch on it, do your work and then send a pull request

**Pull request closed without merging**